content/en/security/application_security/threats/security_signals.md

```diff
@@ -69,11 +146,20 @@ Use [Workflow Automation][5] to manually trigger a workflow for a security signa
 ## Review and remediate
 
 1. On the [Signals Explorer][2] page, select a security signal.
-2. On the signal side panel, click each of the tabs, such as **Attack Flow**, **Activity Summary**, and **Rule Details**, to review the information.
-3. Review the **Suggested Next Steps**, and take action:
+2. On the signal panel, review the attack information.
+3. In **Next Steps**, take action. For example:
     -  Click **Block all Attacking IPs** (by specific duration or permanently).
     -  Click **Automated Attacker Blocking** (based on [detection][10] rules).
-    -  Click **[Block with Edge WAF][11]**.
+    -  Click **[Block with Edge WAF][11]** (if enabled).
```

Comment:
Adding context from Slack. This `Block with Edge WAF` button used to exist, but has since been renamed the Run a workflow button. Might be worth removing this for now, and updating the page specifically for the edge waf blocking initiative (which is being released soon).

cc @nizox 
```suggestion
```